azure-pipelines.yml

```diff
@@ -177,3 +177,19 @@ jobs:
   - script: |
       pip install pytest pytest-azurepipelines pytest-valgrind
       make valgrind
+
+- job: BuildDocs
+  pool:
+    vmImage: 'ubuntu-latest'
+  displayName: 'Build documentation'
+  steps:
+  - template: azure-templates/python.yml
+  - script: sudo apt update && sudo apt install -y libclang-10-dev
+    displayName: 'Install libclang-10-dev'
+  - script: python -m pip install --upgrade pip
+    displayName: 'Upgrade pip'
+  - script: python -m pip install -r docs/requirements.txt
+    displayName: 'Install documentation requirements'
+  - script: |
+      cd docs;
+      python -m sphinx -T -W -E -b html -d _build/doctrees -D language=en . _build/html
```

Comment:
if I understand it correctly, the point of this job is just to check that the docs build, and then we discard the result? If so, maybe add a comment to explain that.
Also, note that as soon as we merge #230 we probably we need re-add this job to the CI

Reply:
Correct. I will add a comment. When we add it back to GitHub actions it might be nice so that one can download the docs from the build for debugging purposes.

I already noted the existence of this PR in #230.

Reply:
Comment added to explain things.